### ISSUE TITLE
NOAA Global GFS Air Temperature Layer Needs Different Defaults

### ISSUE THREAD

**Issue by jamesdoyle21** [asascience/ott-coops#455]
If you load this layer it just covers the entire map in gray because the defaults are set to -5 and 35 deg Kelvin. To be consistent with water tempurature, the units should be in celcius and farenheit, not Kelvin. This also brings up the bigger question of, should we use autoscale as the default range when a user loads a layer?

![image](https://user-images.githubusercontent.com/82955395/164744673-fce2e47a-3904-418e-acdd-877f19e1cce6.png)

**Comment by jamesdoyle21** [comment]
Look at issue #437 with this

**Comment by nguyandy** [comment]
Issue with the units being in Kelvin is a data issue.
https://github.com/asascience/oceansmap-3/issues/151

**Comment by jamesdoyle21** [comment]
@CoryMartinRPS is this a fix on your end?

**Comment by mpiannucci** [comment]
> @CoryMartinRPS is this a fix on your end?

No this is IOOS EDS

**Comment by CoryMartinRPS** [comment]
I don't believe the issue is related to the units, it should be translating to Celcius. However, there is no GFS air temp data this month. I'm looking into it now.

**Comment by mpiannucci** [comment]
> I don't believe the issue is related to the units, it should be translating to Celcius. However, there is no GFS air temp data this month. I'm looking into it now.

Not your fault, this is from IOOS EDS not normal EDS

**Comment by mpiannucci** [comment]
Units are currently in Kelvin on ncwms, need to convert to celsius

**Comment by mpiannucci** [comment]
<img width="936" alt="Screen Shot 2022-06-21 at 9 06 07 AM" src="https://user-images.githubusercontent.com/1832671/174806337-98ac3bd0-9d03-4d85-a3fb-baa2aa0540a2.png">

Need to update the ncml to convert temp to celsius

**Comment by mpiannucci** [comment]
I turned the ncml conversion back on to scale and change the unit, but the default scale and unit from ncwms is still wrong... might take time for the metadata to update or something? @patrick-tripp keeping you in the loop...

<img width="901" alt="Screen Shot 2022-06-21 at 9 16 22 AM" src="https://user-images.githubusercontent.com/1832671/174808466-65afd8e3-c43d-47ff-9c98-5e2c89231abc.png">
